.github/actions/gke/k8s-versions.yaml

```diff
@@ -0,0 +1,22 @@
+# List of k8s version for GKE tests
+---
+k8s:
+  - version: "1.22"
+    zone: northamerica-northeast1-c
```

Comment:
AFAICT we are now spreading to many different zones - are we now more likely to fail the test if any zone has an issue?

Reply:
Yes, and no. These tests only run when it is triggered with a schedule event. So for normal daily runs, it will prefer the same zone as always. With schedule, it will run many clusters at the same time so it would be much more problematic to run them all in the same zone. 
It may be cases where the zone affects the result of a test but also helps to verify if it is related to Cilium or k8s version by isolating the test.